recipes/restinio/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RestinioConan(ConanFile):
+    name = "restinio"
+    license = "BSD-3-CLAUSE"
+    homepage = "https://github.com/Stiffstream/restinio"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "RESTinio is a header-only C++14 library that gives you an embedded HTTP/Websocket server."
+    topics = ("http-server", "websockets", "rest", "tls-support")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"use_boost": [True, False], "use_openssl": [True, False]}
```

Comment:
Can we consider other names and semantic meaning for this options?
- `with_ssl` or just `ssl`, without specifying the TLS implementation used (OpenSSL), like in [paho-mqtt-cpp](https://github.com/conan-io/conan-center-index/blob/5d3df71f13fc04242155bf49f99176d39d95faa2/recipes/paho-mqtt-cpp/all/conanfile.py#L18) or [mysql-connector-c](https://github.com/conan-io/conan-center-index/blob/409aaa96080c7eca837ec243d392048c14b321d0/recipes/mysql-connector-c/all/conanfile.py#L16)?
- `"asio": ["boost", "standalone"]` instead of `"use_boost": [True, False]`, like in [websocketpp](https://github.com/conan-io/conan-center-index/blob/409aaa96080c7eca837ec243d392048c14b321d0/recipes/websocketpp/all/conanfile.py#L15).

Just an ideas to discuss, I don't insist upon it.

Reply:
With_openssl seems more common https://github.com/conan-io/conan-center-index/search?q=with_openssl&type=

These change would be desirable to match CCI